docs/source/introduction/index.rst

```diff
@@ -14,6 +14,7 @@ communications issues.
     getting
     configuring
     communication
+    ext_cfg_util
```

Comment:
While valuable this is specific to the IVI backend so I would move it later (maybe last), or in the FAQ.

Reply:
OK, moved to end.